### PR TITLE
TAJO-1824: Remove partition_keys table from information_schema

### PR DESCRIPTION
**Pull request by blrunner**
Selecting partition keys is not necessary because partitions are searched with its partition name. It would be better to remove from information_schema.

### CODE REVIEW

File: tajo-core-tests/src/test/java/org/apache/tajo/master/TestNonForwardQueryResultSystemScanner.java [hunk 1]
```diff
@@ -38,4 +47,20 @@ public void testGetNextRowsForTable() throws Exception {
   public void testGetClusterDetails() throws Exception {
     assertQueryStr("SELECT TYPE FROM INFORMATION_SCHEMA.CLUSTER");
   }
+
```
Comment (hyunsik) [inline tajo-core-tests/src/test/java/org/apache/tajo/master/TestNonForwardQueryResultSystemScanner.java]: I think that this test is not necessary.

**Comment by hyunsik** [comment]
This change looks straightforward. So, the added unit test is not necessary.

**Comment by blrunner** [comment]
Thanks @hyunsik 

I've just removed the unit test.

**Comment by hyunsik** [comment]
+1

**Pull request closed without merging**